lib/FusionInventory/Agent/Task/Deploy/Job.pm

```diff
@@ -19,6 +20,7 @@ sub new {
         uuid            => $params{data}->{uuid},
         requires        => $params{data}->{requires},
         checks          => $params{data}->{checks},
+        userchecks      => $params{data}->{userinteractions},
```

Comment:
there seems to be a typo here: the left key is 'userchecks', the right one is 'userinteractions'